client/app/scripts/components/node-details/node-details-table-row.js

```diff
@@ -17,22 +22,28 @@ function getValuesForNode(node) {
     }
   });
 
-  (node.parents || []).forEach((p) => {
-    values[p.topologyId] = {
-      id: p.topologyId,
-      label: p.topologyId,
-      value: p.label,
-      relative: p,
+  if (node.parents) {
+    const byTopologyId = groupBy(node.parents, parent => parent.topologyId);
+    const relativesByTopologyId = mapValues(byTopologyId, (relatives, topologyId) => ({
+      id: topologyId,
+      label: topologyId,
+      value: relatives.map(relative => relative.label).join(', '),
       valueType: 'relatives',
-    };
-  });
+      relatives,
+    }));
+
+    forEach(relativesByTopologyId, (columnData, topologyId) => {
+      values[topologyId] = columnData;
+    });
```

Comment:
Doesn't this basically just basically merge `relativesByTopologyId` over values? Would it be any different from simply doing something like `values = {...values, ...relativesByTopologyId}`?